packages/react/src/components/TableCard/TableCard.jsx

```diff
@@ -48,10 +48,10 @@ const StyledStatefulTable = styled(({ showHeader, isExpanded, data, ...rest }) =
       bottom: ${(props) => (!props.isExpanded ? '0px' : '25px')};
       ${(props) => (props.isExpanded ? `width: calc(100% - 50px)` : ``)}
     }
-    .bx--data-table-container {
+    .bx--data-table-content {
       ${(props) =>
         props.data && props.data.length > 0 && !props.isExpanded
-          ? `max-height: 435px;`
+          ? `max-height: 523px;`
```

Comment:
The issue states you are trying to correct for horizontal scroll but these changes seem to be related to height. Does the changing of the class provide the fix?  And are these height changes necessary?

Reply:
The class was incorrect and after having the CSS class fixed. That height was not correct, this is what would render with it:
![image](https://user-images.githubusercontent.com/22034430/126539081-12d54dbb-5707-4c3d-ae9d-adec6b3a6438.png)

So it's kind of another fix than just the scroll. I can update the PR content, sorry about that.